fulu/gp_aug.py

```diff
@@ -18,15 +18,20 @@ class GaussianProcessesAugmentation(BaseAugmentation):
         Example:
             passband2lam  = {0: np.log10(3751.36), 1: np.log10(4741.64), 2: np.log10(6173.23),
                              3: np.log10(7501.62), 4: np.log10(8679.19), 5: np.log10(9711.53)}
+    kernel : kernel object from sklearn
+        Kernel for GaussianProcessRegressor. Can be combine from different kernels.
+        Example:
+            kernel = C(1.0)*RBF([1, 1]) + Matern() + WhiteKernel()
     """
 
     def __init__(self, passband2lam):
-        super().__init__(passband2lam)
+        super().__init__(passband2lam, kernel)
 
         self.ss = None
         self.reg = None
+        self.kernel = C(1.0) * RBF([1.0, 1.0]) + WhiteKernel()
 
-    def fit(self, t, flux, flux_err, passband):
+    def fit(self, t, flux, flux_err, passband, flag_err = False):
```

Comment:
Move `flag_err` to constructor

Reply:
I moved to flag_err to constuctor and made this argument optional, thank you.